src/main.js

```diff
@@ -0,0 +1,16 @@
+import { createCharacter } from "./lib/character.js";
+import { logoSetup } from "./lib/logo.js";
+
+const fetchButton = document.querySelector(".header__search-button");
+
+fetchButton.addEventListener("click", () => {
+  clearCharacters();
+  createCharacter();
+});
+
+function clearCharacters() {
+  const characterElements = document.querySelectorAll(".character");
+  characterElements.forEach((section) => section.remove());
+}
```

Comment:
I would put the `clearCharacters()` function in the `/lib` folder and export it here back. Then you have a beautifully organized `main.js`.